naga/src/front/wgsl/parse/directive/enable_extension.rs

```diff
@@ -126,6 +133,8 @@ pub enum ImplementedEnableExtension {
     ///
     /// [`enable clip_distances;`]: https://www.w3.org/TR/WGSL/#extension-clip_distances
     ClipDistances,
+    /// Enables the `mesh_shader` extension, native only
```

Comment:
thought: I wonder if it'd be an easier diagnostic experience across the board (esp. with bug reports in places not immediately related to wgpu) to have a `naga_*` prefix for Naga-specific extensions.